motor/render/vulkan_renderer.cpp

```diff
@@ -345,6 +345,38 @@ std::vector<vk::ImageView> CreateImageViews(
   return image_views;
 }
 
+vk::RenderPass CreateRenderPass(const vk::Device& vk_dev,
+                                const vk::Format& vk_format)
+{
+  vk::AttachmentDescription color_attachment;
+  color_attachment.setFormat(vk_format)
```

Comment:
could you add a comment saying format should be the same as the one used in swapchain ?